width: 500
height: 500

pixels_per_meter: 75

obstacle_size: 5

min_distance_between_obstacles: 0.1

pub_topic: "map"
background_color: [ 0, 0, 0 ]
obstacle_color: [ 0, 255, 0 ]
sight_cone_color: [ 115, 115, 115 ]

control_color: [ 255, 0, 0 ]
target_color: [ 0, 0, 255 ]

# ORM
# Purple
ORM_goal_color: [ 153, 0, 153 ]
# Yellow
ORM_left_bound_color: [ 51, 255, 255 ]
# Orange
ORM_right_bound_color: [ 0, 128, 255 ]
# Tale
ORM_radius_color: [ 255, 255, 51 ]
# Pink
ORM_radius_security_distance_color: [ 153, 51, 255 ]
ORM_goal_size: 12
ORM_left_bound_size: 12
ORM_right_bound_size: 12
ORM_radius_size: 2.5

sight_cone_range: 2

control_size: 12
target_size: 12

robot_length: 0.4
robot_width: 0.4
robot_color: [ 0, 140, 250 ]

lines_per_meter: 2
line_thickness: 2
meter_line_color: [ 255, 255, 255 ]
